cpp/server/event.cc

```diff
@@ -183,7 +183,7 @@ bool EventLoop::EraseCheck(std::deque<FD*>::iterator* pfd) {
 // static
 void EventLoop::Set(int fd, fd_set* fdset, int* max) {
   DCHECK_GE(fd, 0);
-  CHECK_LT((unsigned)fd, FD_SETSIZE);
+  CHECK_LT((unsigned)fd, (unsigned)FD_SETSIZE);
```

Comment:
Just `unsigned` is really `unsigned int` (because `int` is implicit in C! woo!), so that it's not a fixed type is (mostly) okay.

Now that I think of it, I think that if *all* the casts would be dropped from this line, it might be better. :stuck_out_tongue: